Aerospike_Nozzle/SPEXpdd.tex

```diff
@@ -0,0 +1,292 @@
+				\item Cumulated (thrust) efficiency loss over duration of vehicle's flight path
+				\item 3 main CBN nozzle efficiency losses
+				\item Efficiency losses from a constant external fluid flow
+			\end{itemize}
+		\end{itemize}
+		\item CFD, FEA
+		\begin{itemize}
+			\item Computer simulations will be used to reduce number of designs to experimentally test. They will also serve to improve and validate MATLAB scripts and CAD designs.
+		\end{itemize}
+	\end{itemize}
+	Hardware:
+	\begin{itemize}
+		\item Machined nozzles (smooth-walled)
+		\item Low pressure chamber addition to cold flow test stand
+		\item Apparatus to observe and measure external fluid flow around aerospike
+		\item DAQ system
+	\end{itemize}
+	Events:
+	\begin{itemize}
+		\item Undergraduate Research Symposium
+		\begin{itemize}
+			\item Detailed poster
+			\item Cold flow test stand on display
+		\end{itemize}
+		\item ImagineRIT
+		\begin{itemize}
+			\item Overview poster
+			\item Cold flow test stand on display
+		\end{itemize}
+		\item AIAA Technical Conference
+		\begin{itemize}
+			\item Detailed poster
+			\item Presentation
+		\end{itemize}
+	\end{itemize}
+	Publications:
+	\begin{itemize}
+		\item Monthly SPEX Wiki updates
+		\item Technical paper published in AIAA technical conference format
+	\end{itemize}
+	
+	\subsection{Milestones}
+	\label{subsec:milestones}
+	\textit{(Should I just reference the draft gaant chart found in Appendix A?)}
+	
+	\section{Externalities}
+	% Things not directly related to the work or outcomes, but related to the project as a whole.
+	\subsection{Prerequisite Skills}
+	An overall understanding of compressible flow. Previous experience with MATLAB among at least three of the project members is required. At least one project member must have experience using electronics and sensors.
+	
+	\subsection{Funding Requirements}
+	Excluding CFD and FEA license(s), the cost of all raw materials and DAQ instruments will likely cost \$400 to \$600. If everything is purchased and not donated/sponsored, the most expensive cost would likely be CFD and/or FEA license(s). Fortunately, there have been companies in the past who have donated software licenses, materials, and other equipment to RIT extracurricular groups.
+	
+	\subsection{Faculty Support}
+	Faculty guidance may be needed to help explain some of the concepts this project will explore (e.g. compressible fluid flow, shocks). Guidance may also be needed with DAQ. \textit{(Should specific faculty be listed?)}
+	
+	\subsection{Long-Term Vision}
+	\label{sec:vision}
+	Cultivating an environment that inspires ingenuity is an essential part to SPEX's mission. The predicted efficiency improvement of an aerospike nozzle, combined with its lack of flight heritage, makes it an exciting avenue for research. During this project, the additions made to the cold flow test stand and code written will provide the base for which future nozzle's may be tested and designed using, respectively.
```

Comment:
In the abstract you mention designing an aerospike for use in a sounding rocket--mention that here. What are the stepping stones that will take this project from cold-flow to a 100km rocket nozzle? Rephrasing, what are the main questions to answer in order to take the cold flow designs to be feasible in a 100km sounding rocket?